android-iconify/src/main/java/com/joanzapata/android/iconify/IconDrawable.java

```diff
@@ -208,4 +232,37 @@ public void setStyle(Paint.Style style) {
     	paint.setStyle(style);
     }
 
+    @Override
+    public void setAutoMirrored(boolean mirrored) {
+        if (SDK_INT >= JELLY_BEAN_MR1 && icon.supportsRtl && autoMirrored != mirrored) {
+            autoMirrored = mirrored;
+            invalidateSelf();
+        }
+    }
+
+    @Override
+    public final boolean isAutoMirrored() {
+        return autoMirrored;
+    }
+
+    @TargetApi(KITKAT)
+    private boolean needMirroring() {
+        if (isAutoMirrored()) {
+            // Since getLayoutDirection() is hidden, we will try to
+            // get the layout direction from the View, which we will,
```

Comment:
Taking a second look through the diff, I see that the comma at the end of this line is incorrect. You can remove that if you want.